sdk/core/amqp-common/src/errors.ts

```diff
@@ -128,7 +128,7 @@ export enum ConditionErrorNameMapper {
   /**
    * Error is thrown when timeout happens for the said operation.
    */
-  "amqp:operation-timeout" = "OperationTimeoutError",
+  "amqp:operation-timeout" = "OperationTimeoutError", // Retryable
```

Comment:
If you are adding this comment to `OperationTimeoutError`, then I would expect the same for all the other retryable errors. Otherwise one reading just the comments here would be lead to think this is the only retryable error

Reply:
It actually is on other errors in the list. Is this not the convention?

Reply:
oh cool! I didnt know that :)